packages/core/src/NodeExecuteFunctions.ts

```diff
@@ -480,16 +480,13 @@ export async function parseRequestObject(requestObject: IRequestOptions) {
 	}
 
 	// Axios will follow redirects by default, so we simply tell it otherwise if needed.
+	const { method } = requestObject;
 	if (
-		requestObject.followRedirect === false &&
-		((requestObject.method as string | undefined) || 'get').toLowerCase() === 'get'
-	) {
-		axiosConfig.maxRedirects = 0;
-	}
-	if (
-		requestObject.followAllRedirects === false &&
-		((requestObject.method as string | undefined) || 'get').toLowerCase() !== 'get'
+		(requestObject.followRedirect && (!method || method === 'GET' || method === 'HEAD')) ||
+		requestObject.followAllRedirects
 	) {
+		axiosConfig.maxRedirects = requestObject.maxRedirects;
```

Comment:
Since maxRedirects is optional, shouldn't this fall back to 21? (or another default if we want)

Reply:
nvm I guess axios will still fall back when you pass in `undefined`